.github/workflows/test.yml

```diff
@@ -220,7 +219,12 @@ jobs:
 
       - name: Build
         run: |
-          cargo build -p chain-spec-generator --features fast-runtime --release --locked
+          cargo build -p chain-spec-generator --no-default-features --features fast-runtime,polkadot,coretime-polkadot --release --locked
+
+      - name: Wait for Docker to Start
+        run: |
+          timeout 30 sh -c 'until docker info; do sleep 1; done'
+          docker --version
```

Comment:
may be something like will work?

``` yml
        run: |
          while ! docker info > /dev/null 2>&1; do
            echo "Waiting for Docker to start..."
            sleep 1
          done
          echo "Docker is ready!"
          docker --version
```

Reply:
I can try but it will run forever (means - some CI several-hours-timeout) in case some problem

Reply:
it can be with timeout. Just suggestion